index.js

```diff
@@ -1,6 +1,16 @@
 'use strict';
 
 module.exports = {
+  configs: {
+    recommended: {
+      plugins: ['chai-expect'],
+      rules: {
+        'chai-expect/no-inner-compare': 'warning',
```

Comment:
why did you use `warning` instead of `error` here?

Reply:
Firstly, sorry, it should have been "warn". I've now confirmed against `npm link` that it is reporting (though I have another PR to submit shortly to enhance the rule).

I've also added the docs.

Reply:
And the reason I have "warn" instead of "error" is that one could technically be using it in a valid way but just with bad form, e.g.:

```js
expect(a === 5).to.be.true;
```

Reply:
But I can switch to error if you prefer...

Reply:
yeah, I think `error` would be better.

Reply:
Ok, done. I also updated the README to reflect the defaults completely.